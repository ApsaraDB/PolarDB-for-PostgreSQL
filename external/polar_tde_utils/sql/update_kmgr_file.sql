
create extension polar_tde_utils;

select polar_tde_update_kmgr_file('echo "adfadsfadssssssssfa123123123123123123123123123123123123111313126"');

show polar_cluster_passphrase_command;

select polar_tde_update_kmgr_file('echo "adfadsfadssssssssfa123123123123123123123123123123123123111313123"');

show polar_cluster_passphrase_command;

select kmgr_version_no from polar_tde_kmgr_info_view();

drop extension polar_tde_utils;