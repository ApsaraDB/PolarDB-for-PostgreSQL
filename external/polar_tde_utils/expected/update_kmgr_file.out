create extension polar_tde_utils;
select polar_tde_update_kmgr_file('echo "adfadsfadssssssssfa123123123123123123123123123123123123111313126"');
 polar_tde_update_kmgr_file 
----------------------------
 t
(1 row)

show polar_cluster_passphrase_command;
                    polar_cluster_passphrase_command                     
-------------------------------------------------------------------------
 echo "adfadsfadssssssssfa123123123123123123123123123123123123111313126"
(1 row)

select polar_tde_update_kmgr_file('echo "adfadsfadssssssssfa123123123123123123123123123123123123111313123"');
 polar_tde_update_kmgr_file 
----------------------------
 t
(1 row)

show polar_cluster_passphrase_command;
                    polar_cluster_passphrase_command                     
-------------------------------------------------------------------------
 echo "adfadsfadssssssssfa123123123123123123123123123123123123111313123"
(1 row)

select kmgr_version_no from polar_tde_kmgr_info_view();
 kmgr_version_no 
-----------------
       201912301
(1 row)

drop extension polar_tde_utils;
